src/components/experiment-tracking/runs-list/runs-list.scss

```diff
@@ -1,9 +1,38 @@
 @import '../../../styles/variables';
 
+.runs-list-top-wrapper {
+  position: sticky;
+  top: 0px;
+  z-index: 1;
+}
+
+.search-bar-wrapper {
+  padding-top: 24px;
+  background: var(--color-bg-3);
+  position: sticky;
+  top: 0;
+  z-index: 1;
+}
+
 .runs-list__wrapper {
   width: 100%;
 }
 
 .runs-list__accordion-header {
   padding: 0.8em 3.5em 0.8em 3em;
 }
+
+.compare-switch-wrapper {
+  align-items: baseline;
+  background: var(--color-bg-3);
+  display: flex;
+  justify-content: space-between;
+  padding: 2em 3.5em 1.2em 3em;
```

Comment:
I think this should be `padding: 3.5em 3.5em 1.2em 3em;`. `35px` (plus the little extra from the search component) between the search and compare toggle, not `20px`.